homeassistant/components/cloud/google_config.py

```diff
@@ -477,7 +477,6 @@ def _handle_entity_registry_updated(
 
         self.async_schedule_google_sync_all()
 
-    @callback
     async def _handle_device_registry_updated(
```

Comment:
This can be a callback since there's no await inside.

Reply:
From all comments I always understood that you couldn't combine callback and `async def`

Reply:
Yes. It would just be a callback and and not a coroutine function.

Reply:
Ooooh, then I misunderstood your comment. I thought you meant that in this case it was fine to combine `@callback` and `async def`

Reply:
No, that's never correct.

Reply:
Adjusted